Formula/w/wgo.rb

```diff
@@ -0,0 +1,19 @@
+class Wgo < Formula
+  desc "Watch arbitrary files and respond with arbitrary commands"
+  homepage "https://github.com/bokwoon95/wgo"
+  url "https://github.com/bokwoon95/wgo/archive/refs/tags/v0.5.6d.tar.gz"
+  sha256 "2c2cdfa06678fe58f6d402d6c088a7b4132a717239b74256dbccf1f3421ae7db"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    bin.mkpath
```

Comment:
you dont need this?
```suggestion
```

Reply:
I based this on the `age` formula, so I just took it from there. I'll remove it.

Reply:
oh, i see